software/deploy/deploy.go

```diff
@@ -28,11 +29,14 @@ var (
 )
 
 var (
-	errNoWorkspaceID             = errors.New("no workspace id provided")
-	errNoDomainSet               = errors.New("no domain set, re-authenticate")
-	errInvalidDeploymentID       = errors.New("please specify a valid deployment ID")
-	errDeploymentNotFound        = errors.New("no airflow deployments found")
-	errInvalidDeploymentSelected = errors.New("invalid deployment selection\n") //nolint
+	errNoWorkspaceID                        = errors.New("no workspace id provided")
+	errNoDomainSet                          = errors.New("no domain set, re-authenticate")
+	errInvalidDeploymentID                  = errors.New("please specify a valid deployment ID")
+	errDeploymentNotFound                   = errors.New("no airflow deployments found")
```

Comment:
maybe it should be astronomer deployments? unless they are called airflow deployments in software? in astro we usually say astro deployments

Reply:
It's a pre-existing error message. I don't think we should change it now in this ticket.